### ISSUE TITLE
calender visibility not remembered between browser refreshs

### ISSUE THREAD

**Issue by jasonblewis** [owncloud/calendar#827]
Steps to reproduce
1. log in, notice which calendras are currently visible
2. click on a calendar that is not currently visible on the left hand side to make it visible
3. notice its now visible
4. refresh browser
5. notice calendar is no longer marked as visible.

Also happens in reverse (ie if you mark a visible calendear as not visible, it will be visible again after refresh)

Tested in OwnCloud 10.0.3 (i believe in the previous 9 version i was using before upgrade, it worked as expected)

Also visibility status of calendars is lost on log out/log in.
Tested in Chrome 62.0.3202.75 (Official Build) (64-bit)
and Firefox 52.4.0 (64-bit)

### Expected behaviour
I expect the visible/non visible status of calenders to remain after refresh.

### Actual behaviour
visible/non visible status is lost on refresh
### Server configuration

**Operating system**:
tested in Windows 10 and Linux (Debian Stretch)

**ownCloud version:** (see your admin page)
ownCloud	10.0.3 (stable)
**Calendar version:** (see the apps page)
1.5.4
**Updated from an older installed version or fresh install:**
updated from 9
**Signing status (ownCloud 9.0 and above):**

```
Login as admin user into your cloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.

No errors have been found.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your instance's installation folder

Enabled:
  - activity: 2.3.6
  - calendar: 1.5.4
  - comments: 0.3.0
  - configreport: 0.1.1
  - dav: 0.3.0
  - external: true
  - federatedfilesharing: 0.3.1
  - federation: 0.1.0
  - files: 1.5.1
  - files_external: 0.7.1
  - files_pdfviewer: 0.8.2
  - files_sharing: 0.10.1
  - files_texteditor: 2.2.1
  - files_trashbin: 0.9.1
  - files_versions: 1.3.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 16.0.2
  - market: 0.2.2
  - notifications: 0.3.1
  - provisioning_api: 0.5.0
  - systemtags: 0.3.0
  - templateeditor: 0.1
  - updatenotification: 0.2.1
  - user_external: 0.4
Disabled:
  - encryption
  - files_antivirus
  - theme-example

```

**ownCloud configuration:**

```
Scanning files for 2 users
Starting scan for user 1 out of 2 (Joel)
Starting scan for user 2 out of 2 (jason)

+---------+-------+--------------+
| Folders | Files | Elapsed time |
+---------+-------+--------------+
| 790     | 4956  | 00:00:14     |
+---------+-------+--------------+
debian:/var/www/owncloud# mg config/config.php 
debian:/var/www/owncloud# sudo -u www-data php occ app:list
Enabled:
  - activity: 2.3.6
  - calendar: 1.5.4
  - comments: 0.3.0
  - configreport: 0.1.1
  - dav: 0.3.0
  - external: true
  - federatedfilesharing: 0.3.1
  - federation: 0.1.0
  - files: 1.5.1
  - files_external: 0.7.1
  - files_pdfviewer: 0.8.2
  - files_sharing: 0.10.1
  - files_texteditor: 2.2.1
  - files_trashbin: 0.9.1
  - files_versions: 1.3.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 16.0.2
  - market: 0.2.2
  - notifications: 0.3.1
  - provisioning_api: 0.5.0
  - systemtags: 0.3.0
  - templateeditor: 0.1
  - updatenotification: 0.2.1
  - user_external: 0.4
Disabled:
  - encryption
  - files_antivirus
  - theme-example
debian:/var/www/owncloud# sudo -u www-data php occ config:list system
{
    "system": {
        "instanceid": "xxx",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "owncloud.organictrader.com.au"
        ],
        "datadirectory": "\/var\/lib\/owncloud-data",
        "overwrite.cli.url": "https:\/\/owncloud.organictrader.com.au",
        "dbtype": "mysql",
        "version": "10.0.3.3",
        "dbname": "owncloud",
        "dbhost": "localhost",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "user_backends": [
            {
                "class": "OC_User_IMAP",
                "arguments": [
                    "{imap.organictrader.com.au:993\/imap\/ssl}"
                ]
            }
        ],
        "defaultapp": "calendar",
        "mail_smtpmode": "smtp",
        "mail_from_address": "owncloud",
        "mail_domain": "organictrader.com.au",
        "mail_smtphost": "localhost",
        "mail_smtpport": "587",
        "maintenance": false,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "theme": "",
        "loglevel": 0,
        "appstore.experimental.enabled": true
    }
}



```

**Comment by paulmenzel** [comment]
Could that be a duplicate of issue #812?

**Comment by DeepDiver1975** [comment]
this is fixed with owncloud 10.0.4
